mezzanine/3dtof/files/README.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Files for 3DTof
+permalink: /documentation/mezzanine/3dtof/files/
+---
+## 3DToF Files
+
+Coming soon...
```

Comment:
We should really stop adding lines like these. Or at least give users a deadline for when they should check back...

Reply:
removing the "coming soon" bit since user is never going to be linked to this page anyways. AFAIK we've always just created in README file under every folder. Even if its a folder like this one which is only used the store files that will/are linked to other user facing documents.